app/views/notifications/index.html.erb

```diff
@@ -95,95 +95,43 @@
     </div>
     <div class="articles-list" id="articles-list">
 
-      <section class="notifications-filter__dropdown">
+      <section class="notifications-filter__dropdown s:flex-col">
         <% if @organizations.present? %>
-          <% if @selected_organization %>
-            <div aria-label="notifications-filter-menu">
-              <button type="button" id="notifications-filter__select" class="notifications-filter__dropdown__select">
-                  @<%= @selected_organization.slug %>
-                <%= inline_svg_tag("arrow-down-fill.svg", aria: true, class: "crayons-icon", title: "Arrow Down") %>
-              </button>
-            </div>
-          <% else %>
-            <div aria-label="notifications-filter-menu">
-              <button type="button" id="notifications-filter__select" class="notifications-filter__dropdown__select">
-                  @<%= current_user.username %>
-                <%= inline_svg_tag("arrow-down-fill.svg", aria: true, class: "crayons-icon", title: "Arrow Down") %>
-              </button>
-            </div>
-          <% end %>
-          <div class="notifications-filter__dropdown__menu-overlay" id="notifications-filter__menu-overlay">
-            <nav class="notifications-filter__dropdown__menu" aria-label="notifications-filter-menu">
-              <header class="notifications-filter__dropdown__header">
-                <h3 class="notifications-filter__dropdown__title">
-                  Filter
-                </h3>
-              </header>
-              <a href="/notifications" class="notifications-filter__dropdown__option <%= "selected" if params[:org_id].blank? %>">
-                <div class="notifications-filter__dropdown__label">
-                  <span class="notifications-filter__dropdown__label--title">
-                    <%= @user.name %>
-                  </span>
-                  <span class="notifications-filter__dropdown__label--subtitle">
-                    @<%= @user.username %> · Personal
-                  </span>
-                </div>
-                <div class="notifications-filter__icon">
-                  <%= inline_svg_tag("checkmark.svg", aria: true, class: "crayons-icon", title: "Checkmark") %>
-                </div>
-              </a>
+          <div class="p-2 s:inline-flex">
+            <select onChange="window.location.href=this.value" class="crayons-select">
```

Comment:
I'm a bit uneasy about this.

cc. @nickytonline about it

Reply:
This should be set up using [addEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) via JS. Do you need help with that @ludwiczakpawel? If so, we can sync in the morning/your afternoon.

Reply:
would you be able to help me out with this one?

not really sure how to implement that